### ISSUE TITLE
New Project step: Fix edit button no accessible name

### ISSUE THREAD

**Issue by vyncent-t** [Azure/aks-desktop#200]
This issue an a11y issue related to workflow **Create AKS Project**  https://github.com/Azure/aks-desktop/issues/194

Description:
When following the Create Project workflow in AKS desktop, clicking the "+ Create Project" button at the main tab, then **clicking "AKS Managed Project"** -

**_notice the "Edit" (Pencil) button under the "Project Name" section_** does not have an accessible name. 

Lighthouse scan flagged as the following:

Buttons do not have an accessible name. 

When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

How to test: 

- Navigate to the Projects tab
- Click the "+ Create Projects" button
- Click "AKS Managed Project" option
- Use inspect tools and open Lighthouse
- Scan at this view

---

Lighthouse:

<img width="850" height="641" alt="Image" src="https://github.com/user-attachments/assets/fcc20761-02d8-42ec-a727-0335d8123433" />

View:

<img width="1806" height="990" alt="Image" src="https://github.com/user-attachments/assets/9cfce204-ba97-46da-8f23-a8e70d8c926b" />

<img width="924" height="815" alt="Image" src="https://github.com/user-attachments/assets/9ecaf003-0990-43eb-908a-dc06a9520c99" />
